### ISSUE TITLE
The example app with Exponent cannot be found!!!

### ISSUE THREAD

**Issue by dx51job** [bgryszko/react-native-circular-slider#18]
I've tried to run the example app with Exponent , either by scanning QR code or by typing exp://exp.host/@community/bedtime inside of Expo. 
But unfortunately the example app seems not compatible with the latest version of Exponent（SDK versions 14.0.0+）

**Comment by waxter013** [comment]
Yup. The app need to upgrade from Exponent 12.0.0 to a newer version.
